Source/Resources/Content/MSGraph/DeviceAppManagement/AndroidManagedAppProtections/Protect Policy Managed Client Apps on Unmanaged Android Devices.json

```diff
@@ -273,7 +273,7 @@
   "blockDataIngestionIntoOrganizationDocuments": false,
   "contactSyncBlocked": false,
   "dataBackupBlocked": true,
-  "description": "Configures default behavior for policy managed client apps on unmanaged Android devices.\n\nHardens security for policy managed apps and blocks egress of organization data.",
+  "description": "What is it: Configures default behavior for managed client apps on unmanaged Android devices. Hardens security for policy managed apps and blocks egress of organization data. \n Simeon Notes: Protect your data on users' personal Android devices. This policy allows data to flow between approved apps, but does not allow users to copy or save data outside of the app. Simeon's baseline applies this policy to all users except those in the Azure AD group, 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations' \n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy",
```

Comment:
**What does this do?**
Configures default behavior for managed client apps on unmanaged Android devices. Hardens security for policy managed apps and blocks egress of organization data. 

\n Simeon Notes: Protect your data on users' personal Android devices. This policy allows data to flow between approved apps, but does not allow users to copy or save data outside of the app. Simeon's baseline applies this policy to all users except those in the Azure AD group, 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations' 

**Why should you use this?**
 

**What is the end-user impact?**
 

**Learn more**
[How you can protect app data](https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy)

Reply:
could use preventing instead of blocking egress. Something along the lines of 

"by preventing organization data from leaving approved apps"